src/common/pages/edit/containers/index.js

```diff
@@ -1,17 +1,33 @@
 import { connect } from 'react-redux';
-import Edit from '../components';
-import { onDecrement, onIncrement } from '../store/index';
 
-const mapStateToProps = state => ({
-  counter: state.edit.count,
-});
+import Edit from 'common/pages/edit/components/PasswordForm';
+import addPassword from 'common/api/addPassword';
+import {
+  addPasswordBegin,
+  addPasswordSuccess,
+  addPasswordFailure,
+} from 'common/store/rootReducer';
+
+const addPasswordItem = item => async dispatch => {
+  try {
+    dispatch(addPasswordBegin());
+
+    const response = await addPassword(item); // @todo put real user id
+    const { data } = response;
+
+    dispatch(addPasswordSuccess(data));
+  } catch (e) {
+    dispatch(addPasswordFailure(e));
+
+    throw e;
+  }
+};
 
 const mapDispatchToProps = dispatch => ({
-  increment: () => dispatch(onIncrement()),
-  decrement: () => dispatch(onDecrement()),
+  addPasswordItem: item => dispatch(addPasswordItem(item)),
 });
 
 export default connect(
-  mapStateToProps,
+  undefined,
```

Comment:
We need to use `mapStateToProps` function to get original password's params from the store.